src/components/BaseQuestionnaireResponseForm/widgets/choice/service.ts

```diff
@@ -20,14 +27,15 @@ export async function expandValueSet(answerValueSet: string, searchText: string)
                 ? expandedValueSet.expansion!.contains
                 : [];
 
-            return expansionEntries.map(({ code, system, display }) => ({
```

Comment:
It may be more that just `code, system, display` fields.
That's why we explicitly choose only this three.

Reply:
Done